watertap/costing/watertap_costing_package.py

```diff
@@ -184,6 +184,16 @@ def build_global_params(self):
         )
         self.add_defined_flow("magnesium chloride", self.magnesium_chloride_cost)
 
+        self.phosphorus_recovery_revenue = pyo.Param(
+            mutable=True,
+            initialize=-0.07,
+            doc="Phosphorus recovery revenue",
+            units=pyo.units.USD_2020 / pyo.units.kg,
+        )
+        self.add_defined_flow(
+            "phosphorus salt product", self.phosphorus_recovery_revenue
+        )
+
```

Comment:
same thing here; this should be moved to electronp costing

Reply:
Otherwise this defined flow will be added for other flowsheets that don't have anything to do with phosphorus recovery

Reply:
I moved these variables to unit model